src/rpc/protocol.h

```diff
@@ -64,6 +64,7 @@ enum RPCErrorCode
     RPC_CLIENT_NODE_NOT_CONNECTED   = -29, //!< Node to disconnect not found in connected nodes
     RPC_CLIENT_INVALID_IP_OR_SUBNET = -30, //!< Invalid IP/Subnet
     RPC_CLIENT_P2P_DISABLED         = -31, //!< No valid connection manager instance found
+    RPC_CLIENT_NODE_ADDRESS_INVALID = -32, //!< Node address provided is not valid (too long)
```

Comment:
This is a comment so it's neither a big deal nor a blocker, but I wonder if "too long" is too specific. I could imagine reusing this in other circumstances. Maybe "(e.g. too long)" gives the right intent?

Reply:
removed the "(too long)"